app/api/users_events_roles.py

```diff
@@ -37,10 +39,27 @@ class UsersEventsRolesDetail(ResourceDetail):
     users_events_roles detail by id
     """
 
-    decorators = (api.has_permission('is_coorganizer', methods="GET,PATCH,DELETE"),)
+    def before_delete_object(self, users_events_roles, view_kwargs):
+        """
+        method to check for proper permissions for deleting
+        :param users_events_roles:
+        :param view_kwargs:
+        :return:
+        """
+        if not current_user.is_staff and users_events_roles.role_id == 7:
+            raise ForbiddenError(
+                {'source': 'Role'},
+                'You cannot remove the owner of the event unless you are the admin.',
```

Comment:
Even admin shouldn't be able to remove the owner

Reply:
Ohh okay will update that 😅